app/filtering.js

```diff
@@ -115,12 +115,21 @@ function registerForBeforeRequest (session, partition) {
     }
 
     const firstPartyUrl = module.exports.getMainFrameUrl(details)
+    const url = details.url
     // this can happen if the tab is closed and the webContents is no longer available
     if (!firstPartyUrl) {
       muonCb({ cancel: true })
       return
     }
 
+    if (module.exports.isResourceEnabled('ledger') && module.exports.isResourceEnabled('ledgerMedia')) {
```

Comment:
curious why this was moved

Reply:
also this should abort if `isPrivate` is true

Reply:
added private check